ccxt.js

```diff
@@ -165,6 +165,7 @@ const exchanges = {
     'therock':                 require ('./js/therock.js'),
     'tidebit':                 require ('./js/tidebit.js'),
     'tidex':                   require ('./js/tidex.js'),
+    'tokens':                  require ('./js/tokens.js'),
```

Comment:
Can you please also remove this file from this PR? To do that you just checkout it from master to revert it. This file is generated automatically, so we don't need to commit this edit by hand.